go/fn/.golangci.yml

```diff
@@ -0,0 +1,50 @@
+run:
+  deadline: 5m
+
+linters:
+  disable-all: true
+  enable:
```

Comment:
Why don't we use the default configuration for golangci linters?

Reply:
Some linters don't fit out needs like funlen and stylecheck. So I just used the linter config from kustomize.

Reply:
Actually, those linter errors is from kustomize when I fork kyaml. I'll remove the golangci linters config for now and add it back in the kyaml PR